packages/amazonq/src/lsp/config.ts

```diff
@@ -2,14 +2,34 @@
  * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
  * SPDX-License-Identifier: Apache-2.0
  */
-
+import * as vscode from 'vscode'
 import { DevSettings, getServiceEnvVarConfig } from 'aws-core-vscode/shared'
 import { LspConfig } from 'aws-core-vscode/amazonq'
 
 export interface ExtendedAmazonQLSPConfig extends LspConfig {
     ui?: string
 }
 
+// Taken from language server runtimes since they are not exported:
+// https://github.com/aws/language-server-runtimes/blob/eae85672c345d8adaf4c8cbd741260b8a59750c4/runtimes/runtimes/util/loggingUtil.ts#L4-L10
+const validLspLogLevels = ['error', 'warn', 'info', 'log', 'debug'] as const
+export type LspLogLevel = (typeof validLspLogLevels)[number]
+export const lspLogLevelMapping: Map<vscode.LogLevel, LspLogLevel> = new Map([
```

Comment:
Does this need to be exported?
```suggestion
const lspLogLevelMapping: Map<vscode.LogLevel, LspLogLevel> = new Map([
```